crm_quotation_link/README.rst

```diff
@@ -0,0 +1,52 @@
+CRM Quotation Link
+==================
+
+This module extends the functionality of crm to deliver two contexted buttons into the opportunity form view :
+    the first opens all quotes linked to this opportunity
+    the second opens all sales order linked to this opportunity
+
+Installation
+============
+
+To install this module, you just need to select the module and insure yourself dependancies are available.
+
+Configuration
+=============
+
+No configuration to use this module.
+
+Usage
+=====
+
+To use this module, you need to :
+- create a lead
+- transform it into an opportunity
+- create a quote from this opportunity by using the button
+- you should get back and keep link beetween the models
+
+Known issues / Roadmap
+======================
+
+Contributors
```

Comment:
Contributors section goes under Credits and with `----` style. The same for Maintainer.